### ISSUE TITLE
CT: no item found that has focus (ComboBoxLisgBox)

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/geometric-optics#384]
This has been popping up (pun intended) in geometric-optics, a problem with focus in ComboBoxListBox.  I don't know if it's happening in other sims.  @jessegreenberg @zepumph thoughts?

```js
geometric-optics : phet-io-input-record-and-playback-fuzzBoard : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1647388772654/phet-io-wrappers/input-record-and-playback/?sim=geometric-optics&phetioDebug&fuzzBoard&supportsInteractiveDescription=true&wrapperContinuousTest=%7B%22test%22%3A%5B%22geometric-optics%22%2C%22phet-io-input-record-and-playback-fuzzBoard%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1647388772654%22%2C%22timestamp%22%3A1647391155019%7D
Uncaught Error: Uncaught Error: Assertion failed: no item found that has focus
Error: Assertion failed: no item found that has focus
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1647388772654/assert/js/assert.js:25:13)
at ComboBoxListBox.getFocusedItemNode (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1647388772654/chipper/dist/js/sun/js/ComboBoxListBox.js:245:15)
at Object.keydown (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1647388772654/chipper/dist/js/sun/js/ComboBoxListBox.js:177:75)
at Input.dispatchToListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1647388772654/chipper/dist/js/scenery/js/input/Input.js:1740:23)
at Input.dispatchToTargets (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1647388772654/chipper/dist/js/scenery/js/input/Input.js:1775:12)
at Input.dispatchEvent (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1647388772654/chipper/dist/js/scenery/js/input/Input.js:1698:10)
at Input.dispatchPDOMEvent (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1647388772654/chipper/dist/js/scenery/js/input/Input.js:1040:12)
at Input.keydownAction.PhetioAction.phetioPlayback (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1647388772654/chipper/dist/js/scenery/js/input/Input.js:696:14)
at PhetioAction.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1647388772654/chipper/dist/js/tandem/js/PhetioAction.js:90:17)
at PhetioAction.implementation (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1647388772654/chipper/dist/js/tandem/js/PhetioAction.js:145:18)
id: Bayes Chrome
Snapshot from 3/15/2022, 5:59:32 PM
```

**Comment by jessegreenberg** [comment]
This is not a sim-specific issue but a general problem with headless Chrome and CT. It started showing upbecause we improved testing coverage in https://github.com/phetsims/scenery/issues/1379 recently. But it is happening because certain DOM events that we rely on do not fire in headless Chrome.

`?fuzzBoard` had been effectively disabled on CT until we can address this.

Reassigning to @pixelzoom so he is aware but I think this issue can be closed.

**Comment by pixelzoom** [comment]
👍🏻 closing
